openbas-api/src/main/java/io/openbas/rest/tag/TagApi.java

```diff
@@ -85,15 +111,23 @@ public Tag upsertTag(@Valid @RequestBody TagCreateInput input) {
 
   @Secured(ROLE_ADMIN)
   @DeleteMapping("/api/tags/{tagId}")
-  public void deleteTag(@PathVariable String tagId) {
+  @ApiResponses(value = {@ApiResponse(responseCode = "200")})
```

Comment:
If we return void, should this be a 204 ?

Reply:
Like already discussed in another PR, this sends back a 200 right now.